src/app/models/submission.server.model.ts

```diff
@@ -759,6 +791,7 @@ const buildSubmissionMetadata = (
   result: MetadataAggregateResult,
   currentNumber: number,
   paymentMeta?: PaymentAggregates,
+  mrfMeta?: MultiRespondentAggregates,
```

Comment:
Would recommend to update the function definition to accept optional parameters as an object since I'm spotting a few uses of `foo(paramX, paramY, null, paramZ)`.
```
const buildSubmissionMetadata = (
  result: MetadataAggregateResult,
  currentNumber: number,
  options?: { 
    paymentMeta?: PaymentAggregates,
    mrfMeta?: MultiRespondentAggregates,
  }
)
```

Reply:
refactored to utilise object argument